manifests/install.pp

```diff
@@ -29,11 +29,22 @@
     }
   }
 
-  if $::foreman::passenger and $::foreman::passenger_ruby_package {
-    package { $::foreman::passenger_ruby_package:
-      ensure  => installed,
-      require => Class['apache'],
-      before  => Class['apache::service'],
+  if $::foreman::apache {
+    if $::foreman::passenger {
+      if $::foreman::passenger_ruby_package {
+        package { $::foreman::passenger_ruby_package:
+          ensure  => installed,
+          require => Class['apache'],
+          before  => Class['apache::service'],
+        }
+      }
+    } else {
+      # TODO: foreman-puma subpackage?
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/3315 for RPMs, need one for Debs

Reply:
DEB attempt --https://github.com/theforeman/foreman-packaging/pull/3574

Reply:
So while https://github.com/theforeman/foreman-packaging/pull/3643 was merged, I don't see anything that actually installs Puma in this scenario?

Reply:
https://github.com/theforeman/foreman/blob/develop/bundler.d/service.rb should do this.